raiden/ui/app.py

```diff
@@ -319,8 +359,14 @@ def run_raiden_service(
 
     api_server: Optional[APIServer] = None
     if config.rest_api.rest_api_enabled:
+        assert (
+            config.rest_api.eth_rpc_endpoint
+        ), "An eth_rpc_endpoint should be supplied for the \
```

Comment:
Style nitpick: When it gets too long, please assign the message to a variable and use that in the assert statement.

Also: `should` -> `must`